packages/web-components/src/components/va-privacy-agreement/test/va-privacy-agreement.e2e.ts

```diff
@@ -94,7 +94,8 @@ describe('va-privacy-agreement', () => {
       )
     ).asElement();
 
-    await input.click();
+    input.click();
+    await page.waitForChanges();
```

Comment:
I am not sure if adding `waitForChanges` will help with the test failing in main but it's passing locally and on this branch.

Reply:
:crossed_fingers: